api/kyverno/v1/image_verification_types.go

```diff
@@ -39,6 +39,10 @@ type ImageVerification struct {
 	// See https://github.com/in-toto/attestation. Kyverno fetches signed attestations from the
 	// OCI registry and decodes them into a list of Statement declarations.
 	Attestations []*Attestation `json:"attestations,omitempty" yaml:"attestations,omitempty"`
+
+	// DigestMutate is an optional field which handles the tag-to-digest mutation for the provided image paths.
+	// Defaults to false.
+	DigestMutate bool `json:"digestMutate,omitempty" yaml:"digestMutate,omitempty"`
```

Comment:
@Namanl2001 @samj1912 - should this default to true? 

I am assuming the most common case is that users will want to enable this feature, and hence seems like we should enable by default. Also, that is the current behavior.

In some cases, users may not want Kyverno to mutate the image tag and hence this field allows that additional control.

Thoughts?

Reply:
yes! sounds good to me 👍